src/mlpack/tests/linear_regression_test.cpp

```diff
@@ -83,14 +81,14 @@ BOOST_AUTO_TEST_CASE(ComputeErrorTest)
   // This dataset gives a cost of 1.189500337 (as calculated in Octave).
   LinearRegression lr(predictors, responses);
 
-  BOOST_REQUIRE_CLOSE(lr.ComputeError(predictors, responses), 1.189500337,
-      1e-3);
+  REQUIRE(lr.ComputeError(predictors, responses) ==
+      Approx(1.189500337).epsilon(1e-3 / 100));
```

Comment:
```suggestion
      Approx(1.189500337).epsilon(1e-5));
```
Do you think you could just go with the simpler `1e-5` throughout (or whatever the value should be) instead of including the `/ 100`?

Reply:
Done